lib/ReactViews/WelcomeMessage/WelcomeMessage.jsx

```diff
@@ -49,25 +52,23 @@ const WelcomeMessage = createReactClass({
   }
 });
 
-export const WelcomeMessagePure = ({
-  showWelcomeMessage,
-  setShowWelcomeMessage,
-  isTopElement,
-  viewState
-}) => {
+export const WelcomeMessagePure = props => {
+  return null;
```

Comment:
This kills the welcome message. Intended for now?

Reply:
I'm guessing, since `terria.overrides` isn't re-implemented yet

Reply:
There's no overrides property on terria, so it's broken at the moment. We can revive it if needed in the future.